src/phantom/schema.py

```diff
@@ -35,6 +56,9 @@ def __modify_schema__(cls, field_schema: dict) -> None:
         field_schema.update(
             {key: value for key, value in cls.__schema__().items() if value is not None}
         )
+        if cls.__use_docstring__:
+            if ds := desc_from_docstring(cls):
+                field_schema["description"] = ds
```

Comment:
question: Can we move this to `__schema__`? I'd like to avoid adding logic here.

Reply:
I believe we can't. If the docstring is to take precedence over what overrides of `schema` do, it must happen at the call site of `__schema__`, i.e. `__modify_schema__`, which is what we already discussed in the initial version I pushed.

Even worse, you still might want to  change other fields and thus define `__schema__` to set eg another `title` or add `examples`, but the description should come from the docstring, patched over it. So it must be outside.